messaging/copr_messaging/private/hierarchy.py

```diff
@@ -159,6 +159,21 @@ def package_full_name(self):
             release=release,
         )
 
+    @property
+    def usernames(self):
+        usernames = [self.submitter, self.body.get("owner")]
+        # filter out nones and groups and deduplicate
+        usernames = set(u for u in usernames if isinstance(u, str) and not u.startswith("@"))
+        # return it sorted
+        return list(sorted(usernames))
+
+    @property
+    def groups(self):
+        owner = self.body.get("owner")
+        if isinstance(owner, str) and owner.startswith("@"):
+            return [owner[1:]]
```

Comment:
Just note that the "group"  means Copr group, not FAS group.

Reply:
Oh OK, I thought that copr groups were mapped from FAS groups. Good to know, I'll remove it.

Reply:
They are mapped 1:1, but the names may differ.